docs/release_process.md

```diff
@@ -2,16 +2,42 @@
 
 The release process for AWX is completely automated as of version 19.5.0.
 
+## Get latest release version and list of new work
+
+1. Get latest version of [AWX](https://github.com/ansible/awx/releases)
+2. Get latest version of [AWX Operator](https://github.com/ansible/awx-operator/releases).
+3. [Compare](https://github.com/ansible/awx/compare) `latest` against `devel`
+
+![Comparing Branches](img/compare_release_branch.png)
```

Comment:
It might be clearer if this screenshot showed you selecting an item from the Tags tab.